test/integration/content-preview/ContentPreview.e2e.test.js

```diff
@@ -36,6 +37,36 @@ describe('ContentPreview', () => {
         },
     };
 
+    describe('Sanity', () => {
+        it('The sidebar should not render when sidebar props are omitted', () => {
+            helpers.load({
+                fileId: Cypress.env('FILE_ID_SKILLS'),
+                props: {
+                    contentSidebarProps: {
+                        hasActivityFeed: null,
+                    },
+                },
+            });
+
+            cy.wait(2000); // eslint-disable-line cypress/no-unnecessary-waiting
```

Comment:
Looking to avoid this, but we need to wait until everything is loaded to check for the absence of the sidebar.

Reply:
Since we're checking for a negative, you probably need a positive result first. Can you look for the existence of preview before checking for the absence of the sidebar?

Reply:
unfortunately preview is loading before the sidebar appears when it's turned on. This looks like a potential regression, I would expect the sidebar loading state instantly.

Reply:
looks like the chunk for the sidebar is loading after preview is considered loaded

Reply:
Can we combine the positive and negative case (in that order)? That way we can test for the existence of the sidebar, change the props of ContentPreview, then test for the sidebar's absence?

Reply:
to get the test page to reflect new props, we either have to visit a different page and then visit this one, or call `cy.reload`. Neither of those actions cause the `onbeforeload` callback to be executed, so the component won't re-render based on the new object on the window.

Reply:
Okay. It just seems like having a "sanity" test rely on an arbitrary wait could lead to false alarms. Maybe add a comment explaining the issue.